packages/atlas/src/api/client/index.ts

```diff
@@ -23,6 +25,33 @@ const delayLink = new ApolloLink((operation, forward) => {
   })
 })
 
+const errorLink = onError(({ graphQLErrors, networkError, operation, forward, response }) => {
+  if (graphQLErrors?.some(({ message }) => message === 'Unauthorized')) {
+    forward(operation)
+    return
+  }
+  if (networkError?.message === 'Response not successful: Received status code 400') {
+    forward(operation)
+    return
+  }
+
+  SentryLogger.error(
+    'Apollo Error',
+    'Error Link',
+    {
+      graphQLErrors,
+      networkError,
+    },
+    {
+      data: {
+        operation,
+        response,
```

Comment:
posting this as an object makes it impossible to see orions message (error.response?.data.message) in sentry. Lets either stringify it or include message as a separate field